elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/engine/SQLQueryEngine.java

```diff
@@ -82,6 +83,7 @@ public Schema getSchema(Class<?> entityClass) {
 
     @Override
     public Iterable<Object> executeQuery(Query query) {
+        EntityManager entityManager = emf.createEntityManager();
```

Comment:
entityManager shoudl be closed after each query.  We should put this in a try block so we don't leak it:
```java 
try ( EntityManager entityManager = emf.createEntityManager()) {
   //do stuff
}
```java